internal/journald/journald.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+/*
+Package journald provides functions to read and collect journald logs.
+*/
+package journald
+
+import (
+	"context"
+	"os/exec"
+)
+
+type collectionCommand interface {
+	executeCommand() ([]byte, error)
+}
+
+// Command represents a command that is executed by journalctl.
+type Command struct {
+	ctx     context.Context
+	service string
```

Comment:
In general, context's shouldn't be stored in structs. These two fields are also unused.

Reply:
Thanks for the tip :)